### PR TITLE
[Issue - 484] Dataload GraphExecutionUtil.checkHeaderExist is not able to validate the csv file header

### PR DESCRIPTION
**Pull request by mohammadshahidkhan**
Dataload GraphExecutionUtil.checkHeaderExist is not able to validate the csv file header having header column name in CAPs case as well as with MiXed case

### CODE REVIEW

File: integration/spark/src/test/scala/org/carbondata/spark/testsuite/dataload/TestLoadDataWithHiveSyntax.scala [hunk 1]
```diff
@@ -103,7 +106,18 @@ class TestLoadDataWithHiveSyntax extends QueryTest with BeforeAndAfterAll {
         "'COMPLEX_DELIMITER_LEVEL_1'='$', 'COMPLEX_DELIMITER_LEVEL_2'=':')");
     sql("drop table if exists complexcarbontable")
   }
-  
+
+  test("test carbon table data loading with csv file Header in caps") {
+    sql("drop table if exists header_test")
+    sql("create table header_test(empno int, empname String, designation string, doj String, workgroupcategory int, workgroupcategoryname String,deptno int, deptname String, projectcode int, projectjoindate String,projectenddate String, attendance String,utilization String,salary String) STORED BY 'org.apache.carbondata.format'")
```
Comment (Vimal-Das) [inline integration/spark/src/test/scala/org/carbondata/spark/testsuite/dataload/TestLoadDataWithHiveSyntax.scala]: Add more test cases with create table having column names in upper case and mixed case

**Comment by Vimal-Das** [comment]
Update description, The data loading fails not just for CAPS but for MiXed case as well.

File: integration/spark/src/test/scala/org/carbondata/spark/testsuite/dataload/TestLoadDataWithHiveSyntax.scala (hunk 1 above)
Comment (mohammadshahidkhan) [inline integration/spark/src/test/scala/org/carbondata/spark/testsuite/dataload/TestLoadDataWithHiveSyntax.scala]: added test case with mixed case

**Comment by mohammadshahidkhan** [comment]
Fixed review comments

**Comment by ravipesala** [comment]
@mohammadshahidkhan Please rebase the code

**Comment by mohammadshahidkhan** [comment]
rebase done

**Comment by ravipesala** [comment]
LGTM

**Pull request merged**